_maps/map_files/stations/boxstation.dmm

```diff
@@ -82692,11 +82692,11 @@
 /turf/simulated/floor/plasteel,
 /area/station/maintenance/asmaint)
 "vDY" = (
-/obj/structure/closet/secure_closet/brig,
-/turf/simulated/floor/plasteel{
-	dir = 5;
-	icon_state = "red"
+/obj/effect/turf_decal/delivery/red/hollow,
+/obj/machinery/shower{
+	pixel_y = 8
```

Comment:
south-facing showers should have a much higher y nudge, this is what it looks like with a pixel_y of 16, making it look much more flush with the wall:
![2025_03_25__07_24_47__paradise dme  deltastation dmm  - StrongDMM](https://github.com/user-attachments/assets/ce89c135-3cdd-4684-90ca-ae95177b42f1)